django/tasks/backends.py

```diff
@@ -0,0 +1,119 @@
+from . import constants
+from .base import get_backend
+from .exceptions import ResultUnavailable
+
+
+class TaskResult(object):
+    """
+    Wrapper class representing a status of a task. Provides methods to retrieve
+    the task's status and result value/error.
+    """
+    def __init__(self, backend, task_id):
```

Comment:
Move backend to the back of the argument list and default it to the default backend.